ipc/cli/src/commands/subnet/list_subnets.rs

```diff
@@ -33,11 +33,11 @@ impl CommandLineHandler for ListSubnets {
 
         for (_, s) in ls.iter() {
             println!(
-                "{:?} - status: {:?}, collateral: {:?} FIL, circ.supply: {:?} FIL, genesis: {:}",
+                "{} - status: {:?}, collateral: {} FIL, circ.supply: {} FIL, genesis: {}",
                 s.id.to_string(),
                 s.status,
-                TokenAmount::from_whole(s.stake.atto().clone()),
-                TokenAmount::from_whole(s.circ_supply.atto().clone()),
+                TokenAmount::from_atto(s.stake.atto().clone()),
+                TokenAmount::from_atto(s.circ_supply.atto().clone()),
```

Comment:
What is the type of `s.stake` and `s.circ_supply` here? It comes from the provider, so I assume it's not some Solidity type any more?

Reply:
Good point, they already are token amounts. This conversion is indeed unnecessary now (before Lotus was returning `BigInt` and this is why it was there). Fixed.